code/modules/organs/organ_external.dm

```diff
@@ -1405,7 +1405,7 @@ obj/item/weapon/organ/head/New(loc, mob/living/carbon/human/H)
 		qdel(src)
 		return
 	//Add (facial) hair.
-	if(H.f_style &&  !H.check_hidden_head_flags(HIDEBEARDHAIR))
+	if(H && H.f_style &&  !H.check_hidden_head_flags(HIDEBEARDHAIR))
```

Comment:
Spawning a new head organ by itself was runtiming because its `New()` expected it to always come from a body, so it mostly never checked for the existence of `H`.